tests/distributed/test_pynccl.py

```diff
@@ -0,0 +1,88 @@
+import multiprocessing
+import os
+
+import pytest
+import torch
+
+from vllm.model_executor.parallel_utils.pynccl import (NCCLCommunicator,
+                                                       ncclGetUniqueId)
+
+
+def distributed_run(fn, world_size):
+    number_of_processes = world_size
+    processes = []
+    for i in range(number_of_processes):
+        env = os.environ.copy()
+        env['RANK'] = str(i)
+        env['WORLD_SIZE'] = str(number_of_processes)
+        env['MASTER_ADDR'] = 'localhost'
+        env['MASTER_PORT'] = '12345'
+        p = multiprocessing.Process(target=fn, args=(env, ))
+        processes.append(p)
+        p.start()
+
+    for p in processes:
+        p.join()
+
+
+def update_env(fn):
```

Comment:
add a short docstring for this functionality, i understand this is needed because `Process` constructor doesn't support creating process with env var.

Reply:
fixed in [da58b34](https://github.com/vllm-project/vllm/pull/3625/commits/da58b34b4709da5ebf1fd446714a4b0622f33973).